etna/models/tbats.py

```diff
@@ -83,6 +86,124 @@ def get_model(self) -> Model:
         """
         return self._fitted_model
 
+    def forecast_components(self, horizon: int = 1) -> pd.DataFrame:
+        """Estimate forecast components.
+
+        Parameters
+        ----------
+        horizon:
+            forecast horizon
+
+        Returns
+        -------
+        :
+            dataframe with forecast components
+        """
+        self._check_components()
+        raw_components = self._decompose_forecast(horizon=horizon)
+        components = self._named_components(raw_components=raw_components)
+
+        return pd.DataFrame(data=components)
+
+    def predict_components(self, horizon: int = 1) -> pd.DataFrame:
+        """Estimate prediction components.
+
+        Parameters
+        ----------
+        horizon:
+            forecast horizon
+
+        Returns
+        -------
+        :
+            dataframe with prediction components
+        """
+        raise NotImplementedError("Prediction decomposition isn't currently implemented!")
+
+    def _check_components(self):
+        """Compare fitted model params with the initial params."""
+        fitted_model_params = self._fitted_model.params.components
+
+        seasonal_periods = self._model.seasonal_periods
+        if (
+            seasonal_periods is not None
```

Comment:
Here might be the case where some of the seasonal components are fitted i guess, so we should check which of them are fitted

Reply:
Here we have a general check on component's presence. Seasonal components fitted as specified or not fitted entirely. It do not participate in grid searches as well. 

https://github.com/intive-DataScience/tbats/blob/master/tbats/abstract/Estimator.py#L155